crates/swc_ecma_minifier/tests/projects/output/react-dom-17.0.2.js

```diff
@@ -5537,12 +5531,6 @@
             dispatch
         ];
     }
-    function updateState(initialState) {
```

Comment:
This is actually problematic, it makes output size large.

It's because `initialState` doesn't actually be passed, so `cost_limit` is incorrectly calculated as 3 + 2.